cypress/fixtures/articles_data.json

```diff
@@ -0,0 +1,33 @@
+{
+  "articles": [
+    {
+      "id": 1,
+      "title": "Title1",
+      "sub_title": "sub_title 1",
+      "image": "https://images.unsplash.com/photo-1489710437720-ebb67ec84dd2?ixid=MXwxMjA3fDB8MHxzZWFyY2h8NTZ8fGhhcHB5JTIwcGVvcGxlfGVufDB8fDB8&ixlib=rb-1.2.1&auto=format&fit=crop&w=800&q=60",
+      "content": "Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old.",
+      "author": "journalist@mail.com",
+      "date": "26 december 2020"
+    },
+
+    {
+      "id": 2,
+      "title": "Title2",
+      "sub_title": "sub_title 2",
+      "image": "https://images.unsplash.com/photo-1469793743020-8a607ba773e5?ixid=MXwxMjA3fDB8MHxzZWFyY2h8NzZ8fGhhcHB5JTIwcGVvcGxlfGVufDB8fDB8&ixlib=rb-1.2.1&auto=format&fit=crop&w=800&q=60",
+      "content": "Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.",
+      "author": "journalist@mail.com",
+      "date": "1 december 2020"
```

Comment:
this is not how the date is send from the backend, and you don't get `date` but rather `created_at` `updated_at`